### ISSUE TITLE
Change "CentOS" to e.g. "Rocky Linux" in docs

### ISSUE THREAD

**Issue by koratfood** [netbox-community/netbox#17109]
### Change Type

Cleanup (formatting, typos, etc.)

### Area

Installation/upgrade

### Proposed Changes

As of June 30 2024, CentOS is no longer supported by its vendor/publisher. Should we perhaps update NetBox's docs to reflect this, and replace "CentOS" with e.g. "Rocky Linux" or other non-EoL RHEL-based distribution?

Please feel free to assign this to me if accepted. I can then perform an installation from scratch on whichever RHEL-based distro NetBox prefers, and follow all instructions where CentOS is mentioned, before proposing changes.

**Comment by jeremystretch** [comment]
IMO it might be time to just drop the CentOS-focused portions of the documentation entirely. Most of these are redundant to the documentation for installing externals tools (PostgreSQL, Redis, etc.) anyway.

**Comment by koratfood** [comment]
I disagree that having instructions specific to RHEL-based systems - as opposed to assuming everybody is running Debian-based - is redundant. Removing them from the docs will undoubtedly cause at least some new netbox-community users to struggle with installation unless they already have intimate knowledge of specific differences between RHEL and Debian with regards to names of required packages, manual package post-install commands, and default OS paths + behaviour.

Again, I volunteer to perform the proposed update (if/when accepted), including full testing of every installation step on both Ubuntu and Rocky, and updating/correcting steps where necessary.
